core/signer/signer_test.go

```diff
@@ -0,0 +1,111 @@
+package signer
+
+import (
+	"testing"
+	"time"
+)
+
+const (
+	minDurationS = time.Second * 1
+	maxDurationH = time.Hour * 6
+	minInMax     = int(maxDurationH / minDurationS)
+	fakeNow      = "2015-10-21T04:29:00+00:00"
```

Comment:
On one hand I'd prefer to keep these `const` (which would yield `const initializer parseTime("2015-10-21T04:29:00+00:00") is not a constant`). On the other hand it does increase the readability to initialize the vars as `time.Time` directly. Done in https://github.com/google/keytransparency/pull/598/commits/ce2f58852c917fe884c2c6111bfb61d83abdd382.